Geometry/CMSCommonData/python/cmsExtendedGeometry2017aXML_cfi.py

```diff
@@ -0,0 +1,287 @@
+import FWCore.ParameterSet.Config as cms
+
+## 2015 + new phase 1 pixel detector
+
+XMLIdealGeometryESSource = cms.ESSource("XMLIdealGeometryESSource",
```

Comment:
It does not have changes which are in pre3:
http://cmslxr.fnal.gov/source/Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py#0237
http://cmslxr.fnal.gov/source/Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py#0237
http://cmslxr.fnal.gov/source/Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py#0272
and some others